From a80806ea7b5e84ccac29f495f7cd4e00d08d684b Mon Sep 17 00:00:00 2001
From: Jon West <electrikjesus@gmail.com>
Date: Fri, 22 Feb 2019 23:58:24 -0500
Subject: [PATCH] Add microg fused location provider to config

Looks like we somehow missed this on the commits for Pie

Change-Id: If4dc0b3e7eb4358edb844b80fdd06c739fc1cea3

diff --git a/core/res/res/values/config.xml b/core/res/res/values/config.xml
index 78c13d40230e..a069d16a9779 100644
--- a/core/res/res/values/config.xml
+++ b/core/res/res/values/config.xml
@@ -1684,6 +1684,8 @@
     <string-array name="config_locationProviderPackageNames" translatable="false">
         <!-- The standard AOSP fused location provider -->
         <item>com.android.location.fused</item>
+        <!-- The (faked) microg fused location provider (a free reimplementation) -->
+        <item>com.google.android.gms</item>
     </string-array>
 
     <!-- This string array can be overriden to enable test location providers initially. -->
-- 
2.17.1

